src/ExNavigationStack.js

```diff
@@ -42,7 +42,7 @@ import type { ExNavigationConfig, ExNavigationState } from './ExNavigationTypeDe
 import type { ExNavigationTabContext } from './tab/ExNavigationTab';
 
 const DEFAULT_ROUTE_CONFIG: ExNavigationConfig = {
-  styles: Platform.OS !== 'android' ? NavigationStyles.SlideHorizontal : NavigationStyles.Fade,
+  styles: Platform.OS == 'ios' ? NavigationStyles.Fade : NavigationStyles.SlideHorizontal,
```

Comment:
this does not look right. Fade should be on android.

You probably want to do this:
```
Platform.OS == 'ios' ? NavigationStyles.SlideHorizontal : NavigationStyles. Fade
```